### ISSUE TITLE
scriptRunCommand is not defined

### ISSUE THREAD

**Issue by neil-ji** [open-wc/open-wc#2696]
## Expected behavior

Init the project successfully.

## Actual Behavior

It has throw an error and you can see its details as follow.

√ What would you like to do today? » Upgrade an existing project
√ What would you like to add? » Linting (eslint & prettier), Demoing (storybook), Building (rollup)
√ Would you like to use typescript? » Yes
√ What is the tag name of your web component? ... some-thing

ReferenceError: template:3

    1| {
    2|   "scripts": {
 >> 3|     "storybook": "tsc && <%= scriptRunCommand %> analyze -- --exclude dist && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds -c .storybook/server.mjs\"",
    4|     "storybook:build": "tsc && <%= scriptRunCommand %> analyze -- --exclude dist && build-storybook"
    5|   },
    6|   "devDependencies": {

scriptRunCommand is not defined

## Additional context

I just down the command `npm init @open-wc` and don't know how to find the version of this lib.

**Comment by neil-ji** [comment]
Could any one give me some help or advice?

**Comment by vid** [comment]
I was able to use `npm init @open-wc` last week, but now I'm seeing this too. Weird.

**Comment by neil-ji** [comment]
> I was able to use `npm init @open-wc` last week, but now I'm seeing this too. Weird.

Yes, it is weird and I don't have any clue about it even right now.
I have gave up more trying about this lib. So many disappointment...
